app/Extensions/AppDelegate+Workspaces.swift

```diff
@@ -39,10 +40,27 @@ extension AppDelegate {
         
         configuration.selector.router.didSelectWorkspaceV2 = { [weak self] navigationController in
             { [weak self] workspace in
+                guard let self else { return }
                 didSelect?(navigationController)(workspace)
-                navigationController.viewControllers = [AccountsList.build(navigationController: navigationController)]
-                self?.window?.rootViewController = navigationController
-                self?.window?.makeKeyAndVisible()
+
+                let comingSoonController = ComingSoonViewController(title: "Coming soon..")
+                comingSoonController.view.backgroundColor = UIColor.white
```

Comment:
Please use the design system color and these two lines with the images and tabBarItem belong to the ViewController, please add them there.